### PR TITLE
Feature/show launch screen behind webview

### PR DESCRIPTION
**Pull request by LasseRosenow**
## Summary
This is the new implementation of #1943

After the launch screen the `WebViewActivity` is shown. But the `WebViewActivity` needs some time until the html of the web frontend is loaded. So there is a blank screen between the launch screen and when the frontend is rendered.
To fix this, I changed the background color of the webview to transparent and made the the added the launch screen drawable as the background of the `WebViewActivity`. (On android 12 it will just be a solid color for now).

Also I removed the blue status and navigation bars from the `WebViewActivity` in light mode. This is another step in removing any theme specific colors from the LaunchScreen until the app is drawn and the correct theme color is being extracted from the  web app.

This greatly improves the startup experience of the app and removes some annoying blinking issues when launching.

## Videos
<table>
<tr>
<th>Before</th><th>After</th>
</tr>
<tr>
<td>

https://user-images.githubusercontent.com/10547444/142776946-751ac65e-aae4-43de-8bb2-c46e3d254cdd.mp4

</td>
<td>

https://user-images.githubusercontent.com/10547444/142776947-bbbb3046-7d2f-4eb0-a63d-d572bad2711b.mp4

</td>
</tr>
</table>

### Before
<table>
<tr>
<th>Launch Screen</th><th>WebView</th>
</tr>
<tr>
<td>

before_Moment

</td>
<td>

before_Moment2

</td>
</tr>
</table>

### After
<table>
<tr>
<th>Launch Screen</th><th>WebView</th>
</tr>
<tr>
<td>

before_Moment

</td>
<td>

before_Moment

</td>
</tr>
</table>

## Link to pull request in Documentation repository

Documentation: home-assistant/companion.home-assistant#

### CODE REVIEW

**Comment by LasseRosenow** [comment]
I added some comments explaining the rationale of some of my changes :)

File: app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt [hunk 1]
```diff
@@ -983,78 +990,22 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
...
-                        0 // Remove light flag
-                    } else {
-                        // Theme color is light, so the status bar background should be also light
-                        // Then add the light flag, which indicates that the status bar background is light
-                        WindowInsetsController.APPEARANCE_LIGHT_STATUS_BARS // Add light flag
-                    }
-                    window.insetsController?.setSystemBarsAppearance(flags, WindowInsetsController.APPEARANCE_LIGHT_STATUS_BARS)
-                } else {
-                    var flags = window.decorView.systemUiVisibility
+        println("LOOOL: $statusBarColor")
```
Comment (dshokouhi) [inline app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt:996]: was this left on accident? :)

**Comment by LasseRosenow** [comment]
Sorry for the println I totally forgot about it 😅

**Review by dshokouhi — APPROVED** [review]
Looks good to me!

**Review by JBassett — APPROVED** [review]

**Pull request merged**